packages/workspace/src/browser/workspace-service.ts

```diff
@@ -54,6 +54,10 @@ export class WorkspaceService implements FrontendApplicationContribution {
         this.deferredRoot.resolve(this._root);
     }
 
+    async getDomain(): Promise<string | undefined> {
```

Comment:
Just curious: why did you use the name "domain", for this (I mean in general, not just on this line)?